src/System.Diagnostics.Process/tests/ProcessTests.Unix.cs

```diff
@@ -93,12 +93,7 @@ public void ProcessStart_UseShellExecute_OnUnix_SuccessWhenProgramInstalled(bool
                 using (var px = Process.Start(new ProcessStartInfo { UseShellExecute = true, FileName = fileToOpen }))
                 {
                     Assert.NotNull(px);
-                    if (RuntimeInformation.IsOSPlatform(OSPlatform.OSX))
-                    {
-                        // Assert.Equal(programToOpenWith, px.ProcessName); // on OSX, process name is dotnet for some reason. Refer to #23972
-                        Console.WriteLine($"{nameof(ProcessStart_UseShellExecute_OnUnix_SuccessWhenProgramInstalled)}(isFolder: {isFolder}), ProcessName: {px.ProcessName}");
-                    }
-                    else
+                    if (!RuntimeInformation.IsOSPlatform(OSPlatform.OSX)) // on OSX, process name is dotnet for some reason. Refer to #23972
```

Comment:
Did you intend to include this change?

Reply:
yes. This is a leftover change from one of your comments here: https://github.com/dotnet/corefx/pull/24078

I was going to make this change on a future pr.

Reply:
Best to keep it a separate PR. Sometimes we fix unrelated trivia in a PR, but this is quite distinct and desserves its own.